monitor/xmonitor/indexer/metrics.go

```diff
@@ -46,10 +50,10 @@ var (
 		Buckets:   prometheus.ExponentialBucketsRange(1, 1e6, 10),
 	}, []string{"stream", "xdapp"})
 
-	feesCollectedTotal = promauto.NewCounterVec(prometheus.CounterOpts{
+	feesGweiTotal = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "monitor",
 		Subsystem: "indexer",
-		Name:      "fees_collected_total",
+		Name:      "fees_gwei_total",
```

Comment:
Nit: did you maybe consider `total_fees_gwei` as it's common to use the unit in a suffix?

Reply:
we have a linter that enforces `_total` as suffix for counter metrics

Reply:
yeah, counters are usually suffixed with `total`

Reply:
Oh I see, those are metrics too 👍